CHANGEPLAN.md

```diff
@@ -1,9 +1,64 @@
 # Governance Evolution
 
-## Stages
+## Approach
 
-### Stage 0
+The general approach to transitioning from the existing Cloud Foundry
+Foundation technical governance process to the new process is to start
+with the TOC and then allow that new TOC to work through the details
+of restructuring the community.
 
-### Stage 1
+It will be critically important that existing technical community members,
+and especially technical community leaders, continue their work on the
+projects during the transition process. 
 
-### Stage 2
+We believe that the community will work together with the right intentions 
+during this transition period. When in doubt, community members should
+refer to the newly documented [principles](./PRINCIPLES.md) to guide their
+decisions.
+
+## TOC Formation
+
+### Governing Board Approval
+
+The TOC structure must be approved by the Cloud Foundry Foundation
+Governing Board. Once approved, the technical community can proceed
+with it's reorganization, lead by a newly elected TOC.
+
+### Initial TOC Eligibility
+
+The rules for eligibility to be a member of the TOC include specific
+roles, which do not exist at the time of the first election. As such, 
+eligibility to be nominated for the TOC during the first election will
+be defined as:
+
+* Community members who are PMC Leads, Project Leads or committers within 
+  any PMC or project.
+  
+### Initial TOC Election Voter Eligibility
+
+The rules for eligibility to vote in a TOC election will apply to the 
+first election, but will explicitly include all community members that
+are PMC Leads, Project Leads or committers within any PMC or project.
```

Comment:
Do we expect that any of the mentioned roles will not fulfil the requirements?

Reply:
I think it's possible that they might not be picked up by reporting, but by being explicit here we are 100% sure that the current "maintainer" community is included without question in the first election. It will give us a little breathing room to be sure that the reports we use for defaulting people into eligibility are inclusive enough for the second election cycle.